### ISSUE TITLE
Code links from benchmark page should to go GitHub repo

### ISSUE THREAD

**Issue by dburkhardt** [openproblems-bio/website#40]
Currently, links just go to the current page. E.g.  https://openproblems.bio/benchmarks/batch_integration_graph/

See link location in bottom left of screenshot

<img width="1431" alt="image" src="https://user-images.githubusercontent.com/8322751/197621837-fc92fa6d-adca-4f28-a454-bfbe04e252a5.png">

They should go to the script on GitHub that users can run.

**Comment by scottgigante** [comment]
Link is included in result['Implementation'], e.g. https://github.com/openproblems-bio/website/blob/main/data/results/batch_integration_graph/immune_batch.json#L34

**Comment by rcannood** [comment]
Could it be that the dictionary keys of the results were updated (e.g. there's `result['Implementation']` and `result['Library']`) but the header wasn't (e.g. there is no Implementation or Library in the header, but there is a header for `Code`)?

**Comment by rcannood** [comment]
Also, https://github.com/openproblems-bio/website/blob/main/data/results/batch_integration_graph/immune_batch.json#L34 points to:
```
"Implementation": "https://github.com/openproblems-bio/openproblems/blob/main/openproblems/tasks/_batch_integration/batch_integration_graph/methods/harmony", 
```

However, https://github.com/openproblems-bio/openproblems/blob/main/openproblems/tasks/_batch_integration/batch_integration_graph/methods/harmony does not exist, but https://github.com/openproblems-bio/openproblems/blob/main/openproblems/tasks/_batch_integration/batch_integration_graph/methods/harmony.py does.

**Comment by scottgigante** [comment]
Good point. Will fix the URLs now.

**Comment by rcannood** [comment]
Solved in #137
